### PR TITLE
Rebuilding the image and updating play-with-docker location

### PR DESCRIPTION
**Pull request by azdaly**
Without rebuilding the image, the update does not go through.

### CODE REVIEW

**Comment by azdaly** [comment]
Updating the location of play-with-docker website.

File: docs/tutorial/updating-our-app/index.md [hunk 1]
```diff
@@ -89,9 +89,10 @@ much easier than having to look up the container ID and remove it.
 
 ### Starting our updated app container
 
-1. Now, start your updated app.
+1. Now, rebuild your image and start your updated app.
 
     ```bash
+    docker build -t getting-started
```
Comment (StefanScherer) [inline docs/tutorial/updating-our-app/index.md:95]: The `docker build -t getting-started` step already is in line 22, before the trouble with the used port.

File: docs/tutorial/sharing-our-app/index.md [hunk 2]
```diff
@@ -58,7 +58,9 @@ an example command that you will need to run to push to this repo.
 Now that our image has been built and pushed into a registry, let's try running our app on a brand
 new instance that has never seen this container image! To do this, we will use Play with Docker.
 
-1. Open your browser to [Play with Docker](http://play-with-docker.com).
+1. Open your browser to [Play with Docker](https://www.docker.com/play-with-docker).
```
Comment (StefanScherer) [inline docs/tutorial/sharing-our-app/index.md:61]: If you prefer https only links, then https://labs.play-with-docker.com would also work. Maybe that's easier for users to follow the steps without scrolling and searching for the next link.

File: docs/tutorial/updating-our-app/index.md (hunk 1 above)
Comment (StefanScherer) [reply docs/tutorial/updating-our-app/index.md:95]: I walked through the tutorial and I'm still happy with the current steps. Change the html file, then immediately build the image again and try to run it. The port conflict happens, then we show how to remove the old container and then filly run it again. But that happens all with the rebuilt image.

Is there anything we can improve to make this more visible?


updating-docker-build

Comment (azdaly) [reply docs/tutorial/updating-our-app/index.md:95]: I see that now - I followed the incorrect sequence of steps.  This means that #52 is closed.

**Pull request closed without merging**